doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -548,4 +548,5 @@ cJSON
 Documentation
 =============
 
-|no_changes_yet_note|
+* Removed the Provisioning keys on the nRF54H20 SoC page.
```

Comment:
If this entry is for the 3.2 release, please add a comment on PR - https://github.com/nrfconnect/sdk-nrf/pull/25910. The changelog is cleaned for the next release entries.